doc/tensor.md

```diff
@@ -1327,36 +1331,64 @@ x:indexFill(2,torch.LongTensor{4,2},-10)
 ```
 
 <a name="torch.Tensor.gather"/>
-### [Tensor] gather(src, dim, index) ###
+### [Tensor] torch.gather(src, dim, index) ###
```

Comment:
Sorry, just one more thing :)
Can you remove the "torch."? It's inconsistent with the rest of the docs.